src/content/advanced/gitops/index.md

```diff
@@ -125,367 +123,193 @@ flux-multi-tenancy:
 ```
 
-Install the `Kustomization`:
+This is enough to create the Organization on our own.
 
 ```nohighlight
-kubectl create -f 04-managed-app.yaml
-kustomization.kustomize.toolkit.fluxcd.io/install-managed-app created
-
-kubectl get kustomizations -n default install-organization
-NAME                  READY   STATUS                                                            AGE
-install-managed-app   True    Applied revision: main/070c352f4949ecaad0d43e31744adf7e864cc754   23s
-
-kubectl get apps -n demo0 nginx-ingress-controller-app
-NAME                                VERSION      LAST DEPLOYED   STATUS
-nginx-ingress-controller-app        2.4.0        31s             deployed
+kubectl get namespaces (names will difer)
+NAME            STATUS   AGE
+org-acme   Active   2m29s
 ```
 
-Now the managed application will be installed in your workload cluster. To learn more about how to utilize and configure Managed Apps, please refer to [the documentation]({{< relref "/developer-platform/app-platform/overview">}}).
+### Managing workload clusters
 
-### Watching for new commits
+Next step, once we have an organization is to create workload clusters to run apps.
+In giant swarm there are several providers to create those clusters. The yaml file of the resource is different for every one so we have created some `bases`, templates that help to create the proper configuration for the clusters.
 
-If you have forked the repository, you can test if Flux is watching it for new commits. We are assuming you have completed at least the [Managing organizations](#managing-organizations) section.
+The configuration structures in layer so every upper layer can modify the conifguration and create a custom and very powerful structure. The different possible layers would be:
 
-Let's create a new Organization CR and add it to `02-organization/resources.yaml`
+- Base with fundamental cluster creation manifests
+- Base versions with different modifications
+- Environments that implement config modifications in the bases
+- Clusters that implement specific configuirations
 
-```nohighlight
-echo '---' >> 02-organization/resources.yaml && \
-    kubectl gs template organization --name new-commit >> 02-organization/resources.yaml
-```
+We can create a cluster starting from a base (using different versions) and an environment. The advantages of this are:
 
-Create a commit and push it to your repository's origin.
+- We can group clusters in logical groups that match our infra (dev,pre,prod...)
+- Modifiying the base, we modify all the clusters that implement it (batch update)
+- We can change the clusters between different environments easily
 
-```nohighlight
-git add . && \
-    git commit -m 'add new organization "new-commit"' && \
-    git push
-```
+In this tutorial we are implementing the simpliest form, which is, implement a cluster from a base without applying any extra configuration.
```

Comment:
I re-wrote the term `simplest form` under SOPS, I think this is another term I don't like (pedant of me, I know)

```suggestion
In this tutorial we are implementing a cluster from a base without applying any extra configuration.
```